src/SignalR/server/StackExchangeRedis/src/Internal/RedisProtocol.cs

```diff
@@ -80,21 +81,22 @@ public byte[] WriteGroupCommand(RedisGroupCommand command)
             // * A 'str': The connection Id
             // Any additional items are discarded.
 
-            var writer = MemoryBufferWriter.Get();
+            var memoryBufferWriter = MemoryBufferWriter.Get();
+            var writer = new MessagePackWriter(memoryBufferWriter);
```

Comment:
nit: move this into `try`

Reply:
Done for `var writer = new MessagePackWriter(memoryBufferWriter);`